### PR TITLE
Remove animation names from spritelab

### PR DESCRIPTION
**Pull request by bethanyaconnor**
The names of sprites in spritelab don't really serve a purpose as sprites are referenced by image. In addition the current names are pretty unfriendly (eg `kid_11`). This PR simply removes them from the animation picker dialog:

Screenshot 2020-06-04 at 10 02 10 AM

Category labels are sticking around:
Screenshot 2020-06-04 at 10 02 24 AM

Gamelab still has the animation names. 

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ class AnimationPickerListItem extends React.Component {
             />
           )}
         </div>
-        <div style={labelStyle}>{this.props.label}</div>
+        {this.props.label && <div style={labelStyle}>{this.props.label}</div>}
```
Comment (maddiedierker) [inline apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx:111]: (this isn't the right line to comment on, but github won't let me comment in the proper place)

let's add some vertical padding (probably just paddingBottom) to animations that don't have a label, just to give some more visual separation between them

Comment (bethanyaconnor) [reply apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx:111]: Thoughts? This is 15 px of padding below the animations.

Screenshot 2020-06-04 at 2 24 31 PM

Comment (bethanyaconnor) [reply apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx:111]: For context, I think there's about ~20px of space between the icons if there's a label but that looked a bit much when I tried that.

Comment (maddiedierker) [reply apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx:111]: yeah, that looks great. thank you!

**Review by maddiedierker — APPROVED** [review]

**Pull request merged**